python/svix/api/application.py

```diff
@@ -31,124 +21,237 @@ class ApplicationListOptions(BaseOptions):
     # The sorting order of the returned items
     order: t.Optional[models.Ordering] = None
 
+    def query_params(self) -> t.Optional[t.Dict[str, t.Optional[t.Any]]]:
```

Comment:
```suggestion
    def _query_params(self) -> t.Optional[t.Dict[str, t.Optional[t.Any]]]:
```
Generate these with an underscore (both this and header params) so that it indicates that they are private.